src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/EncryptedXml.cs

```diff
@@ -77,12 +77,15 @@ public class EncryptedXml
         // public constructors
         //
         [RequiresDynamicCode(CryptoHelpers.XsltRequiresDynamicCodeMessage)]
+        [RequiresUnreferencedCode(CryptoHelpers.CreateFromNameUnreferencedCodeMessage)]
         public EncryptedXml() : this(new XmlDocument()) { }
 
         [RequiresDynamicCode(CryptoHelpers.XsltRequiresDynamicCodeMessage)]
+        [RequiresUnreferencedCode(CryptoHelpers.CreateFromNameUnreferencedCodeMessage)]
         public EncryptedXml(XmlDocument document) : this(document, null) { }
 
         [RequiresDynamicCode(CryptoHelpers.XsltRequiresDynamicCodeMessage)]
+        [RequiresUnreferencedCode(CryptoHelpers.CreateFromNameUnreferencedCodeMessage)]
```

Comment:
The ctors themselves don't use dynamic or potentially-unreferenced code.  Why paint them, when all of the affected members seem to also be painted?

Reply:
What I really wanted to do here is annotate the whole class (both `EncryptedXml` and `SignedXml`) as `RequiresDynamicCode` and `RequiresUnreferencedCode`. However, that runs into issues because both of these classes expose `public const string` values, and using these constant values causes warnings from the linker and analyzer in the places that just use the constant strings.

See https://github.com/dotnet/runtime/issues/84433 and https://github.com/dotnet/runtime/issues/81864. Both of those issues were fixed today, but aren't we aren't using those bits in the repo yet.

> when all of the affected members seem to also be painted?

Note that the affected members are suppressing the warnings, not being marked as `RequiresUnreferencedCode`.